core/storage/user/gae_models.py

```diff
@@ -946,6 +950,7 @@ def get_export_policy(cls):
         """Model contains data to export corresponding to a user."""
         return dict(super(cls, cls).get_export_policy(), **{
             'activity_ids': base_models.EXPORT_POLICY.EXPORTED,
```

Comment:
Do not export this.

Reply:
@vojtechjelinek, If I don't export this, there will be an error in `test_all_fields_have_export_policy()` in the `StorageModelsTest` as the lists will not be equal.

Reply:
You can do `'activity_ids': base_models.EXPORT_POLICY.NOT_EXPORTED,`

Reply:
Done.